CMakeLists.txt

```diff
@@ -217,15 +224,20 @@ endif()
 
 if(WITH_VCPKG)
   SET(VCPKG_BASE_DIR "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}")
-  SET(QGIS_PLUGIN_DIR "${VCPKG_BASE_DIR}/tools/qgis/plugins")
   if(MSVC)
+    SET(QGIS_PLUGIN_DIR "${VCPKG_BASE_DIR}/tools/qgis/plugins")
     file(GLOB PROVIDER_LIBS
       "${QGIS_PLUGIN_DIR}/*provider.dll"
     )
+    # From QGIS CMakeLists.txt
+    set(QGIS_PLUGIN_INSTALL_PREFIX "plugins")
   else()
+    SET(QGIS_PLUGIN_DIR "${VCPKG_BASE_DIR}/lib/qgis/plugins")
     file(GLOB PROVIDER_LIBS
       "${QGIS_PLUGIN_DIR}/*provider.so"
```

Comment:
*provider.so -> *provider*.so , and add *authmethod*.so

Reply:
Fixed since master is merged into this branch